modules/tinymce/src/plugins/table/main/ts/ui/ButtonToggleUtils.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+
+import { Singleton } from '@ephox/katamari';
+import Editor from 'tinymce/core/api/Editor';
+import { Toolbar } from 'tinymce/core/api/ui/Ui';
+
+const onSetupToggle = (editor: Editor, styleName: string, styleValue: string) => {
+  return (api: Toolbar.ToolbarMenuButtonInstanceApi) => {
+    const boundCallback = Singleton.unbindable();
+
+    const init = () => {
+      api.setActive(editor.formatter.match(styleName, { value: styleValue }));
```

Comment:
In my rush to get out the PR I forgot to include them.

Reply:
This raises a whole new set of questions then sorry... why aren't we using the `editor.formatter.toggle` API inside the commands then and instead are trying to do the toggling ourselves? The problem is that the button match logic is not the same now for how it actually gets applied. As an example, someone could tweak the `tableclass` format and that would then cause things to break.

Reply:
As a side note, remember we can pass a specific node to `editor.formatter.toggle` like so:
```
editor.formatter.toggle('tableclass', { value: styleValue }, tableNode); 
```
Just mentioning that in case it was done manually since most times the formatter works on the selection instead.